docs/docs/core/08-events.md

```diff
@@ -11,8 +11,8 @@ sidebar_position: 1
 
 There are a few events that are automatically emitted for all messages, directly from `baseapp`.
 
-* `message.action`: The name of the message type.
-* `message.sender`: The address of the message signer.
+* `message.action`: The name of the message type. Exactly one exists for each transaction message, in the first ABCI `Event` of each transaction message.
+* `message.sender`: The address of the message signer. Exactly one exists for each transaction message, in the first ABCI `Event` of each transaction message.
```

Comment:
I think each `in the first ABCI Event of each transaction message.` sentence seems to be a little conflict. Because if all attributes are existed, it can confuse which one should be first according to this sentence. It is a little my opinion. 

And how about moving the additional description as a tip. I think it's nice to explain the default information about each event attribute, and explain more information as a tip.